### ISSUE TITLE
[META] Infrastructure for translating the Backdrop interface into different languages

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop/backdrop-issues#2500]
> As far as I know, the idea to provide a Backdrop translation server was discussed here and there, but not very determined. (...) Besides technical or infrastructure questions I'd like to talk about Backdrop's goals regarding multilanguage capabilities and to have an exchange about how to motivate or activate non English developers or site builders to participate. (from a Gitter conversation)

The above conversation started when I realized that it's actually very difficult to translate the Backdrop interface into another language:

> In the installer we recommend to download a translation file from the (Drupal) translation server and to put in in the files/translations directory. This works fine now as some relevant issues has been resolved. So I thought to get a German site I just have to add new or changed language strings to Backdrop, and that's it. Doing that I noticed that there are really a lot of missing German strings, and now I know why: because of all the Drupal modules which went in Backdrop core, for instance Views. They are of course not covered by the Drupal core translation file!

So @serundeputy suggested to open a meta issue to get help and make progress regarding Backdrop in different languages, which is a good idea in my opinion.

The goals of this issues so far:

- collect existing issues regarding Backdrop localization and translation questions
- identify relevant weaknesses and capabilities
- build a infrastructure to help translating Backdrop
- identify which infrastructure is needed
- motivate people to help translating Backdrop
- help to build language related teams who can work together on translations

(Written down quickly for the moment, I'll change and add some stuff later.)

**Comment by Gormartsen** [comment]
@olafgrabienski I will refer this issue as well here: #1913

**Comment by serundeputy** [comment]
this repo is set up for an MVP candidate for localization: https://github.com/backdrop-ops/localization

We can move any issues related to getting an MVP localization server there.

Down the line or in parallel we can pursue #1913   a better implementation; the idea being not to block or postpone MVP.

**Comment by klonos** [comment]
👍 @olafgrabienski thanx for starting this issue. Sorry I missed the dev meeting today/yesterday guys.

I am very interested in helping with the localization and although my personal need for a multilingual UI and translated websites is not as big as when I was back in Greece, I still would like to see Backdrop work as effortlessly as possible in Greek 😉 @dyrer

**Comment by quicksketch** [comment]
@olafgrabienski has set up a D7 site using the Localization module at https://localize.backdropcms.org/translate/projects/backdropcms

Right now updating the language files on release is a manual process, plus it's another site we'll need to manage accounts on.

Now that we have a dedicated repository, seems like perhaps we should close this issue out and use the general issue queue of that repository. @olafgrabienski made an overall roadmap at https://github.com/backdrop-ops/localization/issues/5.
